modes/vterm/evil-collection-vterm.el

```diff
@@ -265,12 +304,10 @@ But don't allow the cursor to move bellow the last prompt line."
     "P" 'vterm-yank
     "a" 'evil-collection-vterm-append
     "A" 'evil-collection-vterm-append-line
-    "d" 'evil-collection-vterm-delete
```

Comment:
Why are we removing this line?

Reply:
I moved it from 

```lisp
  (evil-collection-define-key 'normal 'vterm-mode-map
```

to

```
  (evil-collection-define-key '(normal visual) 'vterm-mode-map
```

since I was creating this section, and it makes sense for `d` to work in `visual` mode as well.